docker/templates/Dockerfile.erb

```diff
@@ -159,6 +159,9 @@ RUN addgroup -g 1000 logstash && \
   --shell "/sbin/nologin" \
   --no-create-home \
   logstash
+<% elsif %w(full oss).include?(image_flavor)  -%>
+RUN userdel -r ubuntu && groupadd --gid 1000 logstash && \
+  useradd --uid 1000 --gid 1000 --home /usr/share/logstash --no-create-home logstash
```

Comment:
Why the removal of `ubuntu` user is needed?

Reply:
ubuntu 24.04 has an ubuntu user, that wasn't there before, and also has the same uid as we've used for logstash in the past:

```
❯ docker run --rm -it ubuntu:20.04 grep ubuntu /etc/passwd
❯
❯ docker run --rm -it ubuntu:24.04 grep ubuntu /etc/passwd
ubuntu:x:1000:1000:Ubuntu:/home/ubuntu:/bin/bash
```